src/site/content/en/animations/animations-guide/index.md

```diff
@@ -161,22 +126,139 @@ body > .sidebar {
 }
 ```
 
+<!-- TODO(kaycebasques): Link to spec -->
+
 However, the specification suggests this approach should only be taken for elements that are always about to change.
 If the above example was a sidebar the user could slide in and out, that might be the case.
 Some items on your page may not frequently change,
-and so it would be better to apply the property using JavaScript
+and so it would be better to apply `will-change` using JavaScript
 at a point where it becomes likely the change will occur.
-Giving the browser enough time to perform the optimizations needed.
-Then, removing the property once the changing has stopped.
+You'll need to make sure to give the browser enough time to perform the optimizations needed
+and then remove the property once the changing has stopped.
 
 {% Aside %}
 For more information and examples of correct use of `will-change` read
-[Everything You Need To Know About The CSS will-change Property](https://dev.opera.com/articles/css-will-change-property/).
+[Everything You Need To Know About The CSS `will-change` Property](https://dev.opera.com/articles/css-will-change-property/).
 {% endAside %}
 
-The `will-change` property is well supported in browsers.
-If you need a way to force layer creation in older browsers (most likely Internet Explorer at this point)
-then setting `transform: translateZ(0)` will do so.
+If you need a way to force layer creation in one of the rare browsers that doesn't support
+`will-change` (most likely Internet Explorer at this point),
+you can set `transform: translateZ(0)`.
+
+## Debug slow or janky animations {: #debug }
```

Comment:
Trying to focus this on the tasks that the DevTools enable, not sure if we want to phrase it as debugging